stage/pages/lms/dashboard_lms.py

```diff
@@ -0,0 +1,15 @@
+from edxapp_pages.lms.dashboard import DashboardPage
+
+class DashboardPageExtended(DashboardPage):
+
+    def select_course(self):
+        """
+        Selects the course we want to perform tests on
+        Returns:
+
+        """
+        course_names = self.q(css='.course-title a')
+        for vals in course_names:
+            if 'Manual Smoke Test Course 1 - Auto' in vals.text:
```

Comment:
I think you should make this text a variable that you pass into the method. I also think you should raise some exception if the title is not found.

Something like this: 
```python
def select_course_by_title(self, course_title):
  course_names = self.q(css='.course-title a')
  for vals in course_names:
    if course_title in vals.text:
      vals.click()
  raise BrokenPromise("Course title not found") ## <-- no idea if that syntax is correct, but using it as an example
```